chaoslib/openebs/cstor_replica_network_delay.yaml

```diff
@@ -0,0 +1,191 @@
+#- name: Setup pumba chaos infrastructure
+#  shell: >
+#    kubectl apply -f /chaoslib/pumba/pumba_kube.yaml
+#    -n {{ app_ns }}
+#  args:
+#    executable: /bin/bash
+#  register: result
+
+#- name: Confirm that the pumba ds is running on all nodes
+#  shell: >
+#    kubectl get pod -l app=pumba
+#    --no-headers -o custom-columns=:status.phase
+#    -n {{ app_ns }} | sort | uniq
+#  args:
+#    executable: /bin/bash
+#  register: result
+#  until: "result.stdout == 'Running'"
+#  delay: 20
+#  retries: 15
+
+- name: Derive PV from application PVC
+  shell: >
+    kubectl get pvc {{ app_pvc }}
+    -o custom-columns=:spec.volumeName -n {{ app_ns }}
+    --no-headers
+  args:
+    executable: /bin/bash
+  register: pv
+
+- name: Get istgt target pod details
+  #shell: kubectl get pods  -l openebs.io/persistent-volume-claim="{{ APP_PVC }}" --all-namespaces -o custom-columns=:metadata.name
+  shell: kubectl get pods  -l openebs.io/persistent-volume-claim="percona-mysql-claim" --all-namespaces --no-headers -o custom-columns=:metadata.name
+  register: istgt_replica
+
+#kubectl get pods -n app-percona-ns  -l 'name=percona'  --no-headers -o custom-columns=:metadata.name
+- name: Get Application  pod details
+  #shell: kubectl get pods  -l openebs.io/persistent-volume-claim="{{ APP_PVC }}" --all-namespaces -o custom-columns=:metadata.name
+  shell: kubectl get pods -n {{ app_ns }}  -l {{ APP_LABEL }}  --no-headers -o custom-columns=:metadata.name
+  register: app_pod
+
+- name: Get list of cstor replicas pod belonging to the PV
+  shell: >
+    kubectl get pods -l app=cstor-pool
```

Comment:
this will give all cstor-pools.. but, we want just those that are related to this PVC

Reply:
https://github.com/openebs/litmus/pull/372/files you can follow this to find pools

Reply:
ok